src/soap/create-folder-request.ts

```diff
@@ -16,7 +16,7 @@ export const createFolderRequest = async ({
 		_jsns: 'urn:zimbraMail',
 		folder: {
 			color,
-			f: excludeFreeBusy ? 'b' : '',
+			f: excludeFreeBusy ? 'b' : '#',
```

Comment:
You should ask validators if they would like it checked as default even if it is excluded from freebusy

Reply:
ive Aligned with Eugenia and i changed it, ty!